x/expiration/types/params.go

```diff
@@ -0,0 +1,98 @@
+package types
+
+import (
+	"fmt"
+	"time"
+
+	"gopkg.in/yaml.v2"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+const (
+	NhashDenom = "nhash"
+)
+
+// DefaultDuration is the default duration a module asset
+// will live on chain before expiring. Defaults to 1 year
+var DefaultDuration = 24 * 365 * time.Hour
+
+var DefaultDeposit = sdk.Coin{
+	Denom: NhashDenom,
+	// todo: what should this be? should this be a configurable property in app.toml?
+	Amount: sdk.NewInt(1905),
+}
```

Comment:
1) Should probably use `sdk.NewIng64Coin(NhashDenom, 1905)`
2) It should probably be something other than `1905` since that's the gas price, and this has nothing to do with that, and is probably way too low.
3) Unaddressed TODO. And Yeah. The denom should come from the config. We'll need to discuss the amount, but I feel like it'd be closer to `1000000000nhash` (1 hash). Alternatively, we could just define it as `0nhash` here so that it's only meaningful if set in params.

Reply:
I set this as `1000000000nhash`.

> The denom should come from the config.

If this value is moved to the config, would that not lead to inconsistent state when set to a different value from node to node? I am not certain on this but it may also effect `state sync`.

Reply:
This has been updated to use `sdk.NewInt64Coin(NhashDenom, 1000000000)`

Reply:
Changed to default sdk.NewInt64Coin(NhashDenom, 0) it's only meaningful when set in params.